src/plugins/application_config/server/routes/index.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  IRouter,
+  IScopedClusterClient,
+  Logger,
+  OpenSearchDashboardsResponseFactory,
+} from '../../../../core/server';
+import { ConfigurationClient } from '../types';
+
+export function defineRoutes(
+  router: IRouter,
+  getConfigurationClient: (inputOpenSearchClient: IScopedClusterClient) => ConfigurationClient,
+  logger: Logger
+) {
+  router.get(
+    {
+      path: '/api/config/exists',
+      validate: false,
+    },
+    async (context, request, response) => {
+      const client = getConfigurationClient(context.core.opensearch.client);
+
+      return await handleExistsConfig(client, response, logger);
+    }
+  );
+}
+
+export async function handleExistsConfig(
+  client: ConfigurationClient,
+  response: OpenSearchDashboardsResponseFactory,
+  logger: Logger
+) {
+  try {
+    const result = await client.existsConfig();
+    return response.ok({
+      body: {
+        exists: result,
+      },
```

Comment:
any reason we wrap output within body?

Reply:
Removed the API. However, I originally added the `body` due to existing examples in OSD code base

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/eff7cb5536a552692c43fedd60fb8a9f87f1e801/examples/routing_example/server/routes/message_routes.ts#L97